### PR TITLE
Fixed #48483

### LINKED ISSUE #48483: Flash of Unstyled content using Ant Design with Nextjs
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.2.0: Fri Nov 11 02:08:47 PST 2022; root:xnu-8792.61.2~4/RELEASE_X86_64
    Binaries:
      Node: 16.15.0
      npm: 8.5.5
      Yarn: 1.22.18
      pnpm: N/A
    Relevant packages:
      next: 13.3.1-canary.8
      eslint-config-next: 13.3.0
      react: 18.2.0
      react-dom: 18.2.0
```

### Which example does this report relate to?

with-ant-design

### What browser are you using? (if relevant)

Chrome 112.0.5615.49	

### How are you deploying your application? (if relevant)

Vercel

### Flash of Unstyled content using Ant Design with Nextjs :
When using Ant Design with Next JS 
the HTML gets rendered unstyled ( barebone HTML elements) for a fraction of a second , and then the styles get loaded and the components are rendered correctly

### Expected Behavior

the first contentful-paint should be properly styled

### To Reproduce

clone the with-ant-design example and host it

### PR DESCRIPTION
**Pull request by SamarthBagga**
### What

Fixed issue #48483- There was a flash of unstyled design in the with-ant-design example in the repo.

### How

Created a _document.tsx file with a head component with a link to the ant design stylesheet.

### Why

To prevent the flash of unstyled content in my Next.js application, I added a custom Document component. In this component, I included a link to the Ant Design CSS stylesheet within the <Head> section. This ensures that the styles from Ant Design are loaded and applied to the page during server-side rendering (SSR). By extending the default Document component, I was able to customize the HTML structure of the page and include the necessary styles, providing a seamless and styled user experience.

### CODE REVIEW

File: examples/with-ant-design/pages/index.tsx [hunk 1]
```diff
@@ -26,6 +28,10 @@ export default function Home() {
   }
 
   return (
+<div>
+  <Head>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/antd/dist/antd.css" />
```
Comment (B2o5T) [inline examples/with-ant-design/pages/index.tsx:33]: https://nextjs.org/docs/messages/no-stylesheets-in-head-component#document

Comment (SamarthBagga) [reply examples/with-ant-design/pages/index.tsx:33]: thanks for the heads up ! I'll make the necessary changes.

Comment (SamarthBagga) [reply examples/with-ant-design/pages/index.tsx]: Hi @B2o5T 
I have made the changes according to your suggestion.

**Comment by SamarthBagga** [comment]
Hi @B2o5T could you review my pull request?

**Comment by leerob** [comment]
Closing this out, as we'd like to move the example to use the App Router instead 🙏

**Pull request closed without merging**